core/lib/via_btc_client/examples/fee_history.rs

```diff
@@ -0,0 +1,49 @@
+use std::{env, str::FromStr};
+
+use anyhow::{Context, Result};
+use tracing::info;
+use via_btc_client::{
+    inscriber::Inscriber,
+    types::{BitcoinNetwork, NodeAuth},
+};
+
+const RPC_URL: &str = "http://0.0.0.0:18443";
+const RPC_USERNAME: &str = "rpcuser";
+const RPC_PASSWORD: &str = "rpcpassword";
+const NETWORK: BitcoinNetwork = BitcoinNetwork::Regtest;
+const PK: &str = "cRaUbRSn8P8cXUcg6cMZ7oTZ1wbDjktYTsbdGw62tuqqD9ttQWMm";
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    tracing_subscriber::fmt()
+        .with_max_level(tracing::Level::INFO)
+        .init();
+
+    let args: Vec<String> = env::args().collect();
+    let number_blocks = usize::from_str(&args[1].to_string())?;
+
+    let inscriber = Inscriber::new(
```

Comment:
maybe we can create BtcClient without inscriber? is there any reason we should use inscriber here?